recipe/meta.yaml

```diff
@@ -1,25 +1,30 @@
 {% set name = "jupyterlab-latex" %}
-{% set version = "3.1.0" %}
+{% set version = "4.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyterlab_latex/jupyterlab_latex-{{ version }}.tar.gz
-  sha256: f44df285405b5b552b70d10f1f0573d61ca3a4d2ca50bd3097a63a27cdcde16e
```

Comment:
Please update the hash to validate the content of the package does not change.

Reply:
Done, tks!